pkg/gameplay/meta_events.go

```diff
@@ -188,6 +191,7 @@ func HandleMetaEvent(ctx context.Context, evt *pb.GameMetaEvent, eventChan chan<
 			return ErrNotAllowed
 		}
 
+		// XXX: Adjust reasonably based on receiver's remaining time.
```

Comment:
it won't really matter for the time being; game would end before nudge/abort times out so i think it's fine.